### PR TITLE
Fix a search and refactor ui visibility

### PR DESCRIPTION
**Pull request by santaasus**
1) It’s overhead call a new SearchTask().executeOnExecutor an every
time.Therefore, I added some callback, timer’s delaying with 200
millisecond.When an user is stop writing it’s get list of songs.
2) & ~ == XOR

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -135,7 +135,8 @@
     <string name="play_next">Play next</string>
     <string name="new_playlist">New playlist</string>
 
-
+    <string name="fabric_api_key">12345</string>
+    <string name="play_billing_license_key">12345</string>
```
Comment (Kasijjuf) [inline app/src/main/res/values/strings.xml]: These fake API keys should not be here.

**Comment by Kasijjuf** [comment]
Unless this change requires incrementing the SDK version, the SDK increase should be it's own pull request.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (santaasus) [inline app/src/main/res/values/strings.xml]: Without these ones CI throw error

**Comment by naman14** [comment]
I don't think this is really needed, there is no considerable delay in searching on each character change

**Pull request closed without merging**